README.md

```diff
@@ -1,94 +1,46 @@
 # A Reference Process for OSS Projects on GitHub
 
-This is an attempt to document similarities in in a number of our OSS projects hosted on GitHub. 
+This is an attempt to document practices common to a number of OSS projects we have on GitHub. 
```

Comment:
BTW, I'm open to change `OSS-Generic` to a more suitable name (as long as it doesn't have `GitHub` or `NUS` in it). It's just that I couldn't think of a better one at the time.